modules/ROOT/pages/sap/sap-connector.adoc

```diff
@@ -93,7 +93,7 @@ Java API::
 Handles dynamic metadata lookup and caching. The Java API implements the `JCO.Function` container used for parameters and/or tables for the SAP BAPI function module in Java. Java apps are built on top of the Java API.
 
 JNI (Java Native Interface)::
-Originally, SAP created libraries in C language to allow direct RFC calls to SAP, to manipulate with data. JCo wraps C libraries in Java to provide platform-native access into the SAP system. RFC middleware uses the RFC library through the JNI Layer.
+Originally, SAP-created libraries in C language to allow direct RFC calls to SAP, to manipulate with data. JCo wraps C libraries in Java to provide platform-native access into the SAP system. RFC middleware uses the RFC library through the JNI Layer.
```

Comment:
I don't understand what the first sentence is trying to communicate. Is it this?

Originally, SAP-created libraries in C language allowed direct RFC calls to SAP to manipulate data.

I would also remove the comma after SAP